privacyidea/static/components/config/views/config.events.details.html

```diff
@@ -14,7 +14,8 @@
         Enable
     </button>
     <button class="btn btn-danger"
-        ng-click="delEvent(eventid)">
+            ng-click="confirmDelete(delEvent, eventid,
+            'Do you really want to delete this event?')">
```

Comment:
I am so sry.

One last nitpicking: It would be great to make this translatable:

I think it should look like this:
~~~~html
<button ....  
ng-click="confirmDelete(delEvent, 
   eventid, 
   {{ 'Do you really want to delete this event?' | translate }})">
~~~~

Reply:
the translation is already implemented in the JS method.

~~~~js
  $scope.confirmDelete = function(delete_function, identifier, question) {
        if ($scope.deletion_confirmation) {
if (confirm(gettextCatalog.getString(question))) {
                delete_function(identifier);
            }
        } else {
            delete_function(identifier);
        }
    };
   {{ 'Do you really want to delete this event?' | translate }})">
~~~~

Reply:
Have you tried to call s.th. like `gettextCatalog.getString("Do you really want to delete this resolver?")` directly in the html templates?

Having this string array really looks nasty and is probably prone to errors.